### ISSUE TITLE
#9 ContactCard

### ISSUE THREAD

**Issue by manukao** [manukao/interactive-portfolio-project-1#16]
User Story:
As a user of the portfolio website,
I want to be able to get in touch with the developer
to ask questions or request a collaboration.

Acceptance Criteria:

- [x] Display a contact form containing at least a name/company & text input field
- [x] Submitting the form will send the users input to the developer

Tasks:

- [x] Create a ContactCard component to render the contact form
- [x] Create a function that sends the users input to the developer
- [ ] (consider using TypeScript for this component)
- [x] All acceptance criteria are covered by component testing

![Image](https://user-images.githubusercontent.com/97320894/231210298-dbe3be70-7f62-41c3-b61c-22e33e8ad63e.png)

**Comment by alexWaligorski** [comment]
Hi Manu! How great that you are creating your own website as a project. Super practical to have one after the bootcamp.
A contact form makes a lot of sense but I would have the following suggestions/ remarks for your userstory:

- What will be the content of the contact form? Which inputs will it have? Your wireframe could also be more specific about the structure of the form
- Maybe be more specific about the "message" that will be send to the developer: "The form Data will be send to the developer" or  will the developer be only notified that some data e.g. a comment has been left by a vistor of his/her website?
- Why do you need a TestimonialsCard and not a ContactCard?
- You should add create a form component to your tasks
- I think you can leave out "consider usig typescript".. This is not a task but a consideration you can make or not.. it would not be necessary to create the form
- I would be more specific about "Find a way to grab..." and rather word it like "Send the form data to page owner / dveloper"

I hope this helps!

**Comment by manukao** [comment]
Hi Alexandra, thanks for your feedback 👍 

"TestimonialsCard" was a copy/paste typo from my last US ;) thanks for pointing that out! Fixed that.

As for the rest, I intended to keep the US as open as possible but still concise enough as a contact form usually always is kind of the same (with name/email/text input or something similar) while there are a lot of possible solutions for this problem and I did not yet decide what I will do. However, I added "... containing at least a name/company & text input field"

I only specified in the acceptance criteria and also in the tasks that the input should be **sent** to the developer as opposed to for example "only" triggering the users default email app.

Using TypeScript is just a reminder for myself - put it in brackets now :-)

Kind regards
